docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -551,6 +563,24 @@ Note: By default, the sum of traffic is rounded to 100. If both `primary` and `c
 | duration | duration | Maximum time to perform the analysis. | Yes |
 | metrics | [][AnalysisMetrics](#analysismetrics) | Configuration for analysis by metrics. | No |
 
+## PostSync
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| chain | [DeploymentChain](#deploymentchain) | Deployment chain configuration, used to determine and build deployments that should be triggered once the current deployment is triggered. | No |
+
+### DeploymentChain
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| applications | [][DeploymentChainApplication](#deploymentchainapplication) | The list of applications which should be triggered once deployment of this application rolled out successfully. | Yes |
+
+### DeploymentChainApplication
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| name | string | The name of PipeCD application, note that application name is not unique in PipeCD datastore | No |
```

Comment:
At least `name` and `kind` should be allowed at the first release of the Deployment Chain feature since `name` only is not enough for using.

Reply:
Sure, that's way better 👍 PTAL https://github.com/pipe-cd/pipe/pull/2934